Files/View Models/ItemViewModel.cs

```diff
@@ -1251,11 +1252,14 @@ private void AddFolder(WIN32_FIND_DATA findData, string pathRoot)
                 systemTimeOutput.Milliseconds,
                 DateTimeKind.Utc);
             var itemPath = Path.Combine(pathRoot, findData.cFileName);
+            string itemName = string.Empty;
+            if (!AppSettings.ShowLocalisedName) itemName = findData.cFileName;
+            else  // ItemName will be loaded later to improve perfomance
```

Comment:
You can fill the name field with the untranslated name. It will automatically update ones the LoadExtendedItemProperties is called. Might look much better that an empty name.